src/index.js

```diff
@@ -4,12 +4,15 @@
  * @returns empty object
  */
 const validateEvent = (event) => {
+    console.log('Validating event...')
+
+    console.log('Event validated!')
     // Dummy function for design
     return{}
 }
 
 
-export const handler = async (event) => {
+module.exports.handler = async (event) => {
```

Comment:
Had to revert to non ES6 syntax. I couldn't get VS code to work with it. I will keep looking into it though.